src/components/version-switcher/version-switcher.module.css

```diff
@@ -28,7 +28,8 @@ https://app.asana.com/0/1202097197789424/1204412156894162/f
 
 	/* Make dropdown list fill container */
 	& > div {
-		width: 100%;
+		min-width: 100%;
+		width: max-content;
```

Comment:
This fixes an issue where the `VersionSwitcher` [listPosition](https://github.com/hashicorp/dev-portal/blob/zs.openapi-versioned-docs/src/components/version-switcher/index.tsx#L39) being passed to our dropdown disclosure component wasn't working as expected (due to interactions with these "overrides).

Note this change fixes _all_ uses of version switcher, not just the use in `OpenApiDocsView`.

| Before | After |
| - | - |
| ![version-switcher-before](https://github.com/hashicorp/dev-portal/assets/4624598/1c3742d2-0a70-43fa-93ee-8bd48d5fc1cf) | ![version-switcher-after](https://github.com/hashicorp/dev-portal/assets/4624598/93144eef-9924-4399-99b2-e5abc2039c64) |

Reply:
Oh awesome! nice find.